buildpacks/ruby/src/steps/default_env.rs

```diff
@@ -0,0 +1,41 @@
+use libcnb::{build::BuildContext, data::layer_name, layer_env::Scope, Env};
+
+use crate::{
+    layers::{EnvDefaultsSetSecretKeyBaseLayer, EnvDefaultsSetStaticVarsLayer},
+    RubyBuildpack, RubyBuildpackError,
+};
+
+pub struct DefaultEnv;
+
+impl DefaultEnv {
+    pub fn call(
```

Comment:
Agreed. We talked about it in one meeting and there was one really good idea for naming so that they stood out. But I can't remember what it was.

I went ahead and renamed them. I'm referencing them by full path in my main so they stand out.